http/src/request/channel/invite/delete_invite.rs

```diff
@@ -1,9 +1,16 @@
 use crate::request::prelude::*;
+use twilight_model::invite::Invite;
 
 /// Delete an invite by its code.
+///
+/// Requires the [`MANAGE_CHANNELS`] permission on the channel this invite
+/// belongs to, or [`MANAGE_GUILD`] to remove any invite across the guild.
+///
+/// [`MANAGE_CHANNELS`]: twilight_model::guild::Permissions::MANAGE_CHANNELS
+/// [`MANAGE_GUILD`]: twilight_model::guild::Permissions::MANAGE_GUILD
 pub struct DeleteInvite<'a> {
     code: String,
-    fut: Option<Pending<'a, ()>>,
+    fut: Option<Pending<'a, Invite>>,
```

Comment:
This is a breaking change, how about we do this individual change in `next`? We have precedent with https://github.com/twilight-rs/twilight/pull/758